x-pack/plugins/maps/common/descriptor_types/style_property_descriptor_types.ts

```diff
@@ -36,6 +37,7 @@ export type LabelBorderSizeStylePropertyDescriptor = {
 export type FieldMetaOptions = {
   isEnabled: boolean;
   sigma?: number;
+  percentiles?: number[];
 };
 
 export type StylePropertyField = {
```

Comment:
+1 this feels like a minimal approach to add this functionality, without re-architecting the related descriptors.